lutris/gui/config/boxes.py

```diff
@@ -446,8 +451,13 @@ def generate_directory_chooser(self, option, path=None):
         self.option_widget = directory_chooser
 
     def _on_chooser_dir_set(self, entry, option):
-        """Action triggered on file select dialog 'file-set' signal."""
-        self.option_changed(entry.get_parent(), option, entry.get_text())
+        """Action triggered when the field's content changes."""
+        text = entry.get_text()
+        if text.startswith('file:///'):
+            text = text[len('file://'):]
```

Comment:
I don't buy bit; this does not handle URL escaping ,like file:///dir/file%20with%20spaces -> /dir/file with spaces

I could fine no simple way to handle decoding a file URL, but apparently you can do something with urllib.parse.unquote.

Reply:
Interesting, it works on my machine even without the unquoting. But yes, unquoting would not hurt.

I have updated my commit

Reply:
That seems very strange, but I expect the new version will be safer.